src/psyclone/tests/test_files/dynamo0p3/testkern_stencil_region_mod.f90

```diff
@@ -31,10 +31,9 @@
 ! POSSIBILITY OF SUCH DAMAGE.
 ! -----------------------------------------------------------------------------
 ! Author R. W. Ford, STFC Daresbury Lab
-! Modified I. Kavcic, Met Office
+! Modified I. Kavcic, A. Coughtrie, Met Office
```

Comment:
Please update the copyright year range.

Reply:
Done, see commit fdd34d286c25ea21986ea3520f7ae3d039b2937e